README.md

```diff
@@ -73,6 +73,7 @@ pnpm db:seed
 For a complete reference of all available D1 database commands, see the [Cloudflare D1 Wrangler Commands documentation](https://developers.cloudflare.com/d1/wrangler-commands/).
 
 3. Create a `.env` file in the root directory with the following variables (replace values with your own):
+
 ```sh
```

Comment:
Setting the app up I only needed `CLOUDFLARE_DATABASE_ID`.. I think

Reply:
account id probably matters for different commands

Reply:
like deploy haha